CMakeLists.txt

```diff
@@ -332,6 +332,7 @@ include_directories(${PROJECT_SOURCE_DIR}/include)
 add_subdirectory(src)
 
 if (BENCHMARK_ENABLE_TESTING)
+  find_package(Python3 3.6 REQUIRED COMPONENTS Interpreter)
```

Comment:
i read a bit about find_package but couldn't find my answer: i know this isn't an EXACT match but what versions of python3 will this match? should it be a version range?